astronomer/providers/dbt/cloud/operators/dbt.py

```diff
@@ -0,0 +1,72 @@
+import time
+from typing import Any, Dict
+
+from airflow import AirflowException
+from airflow.providers.dbt.cloud.hooks.dbt import DbtCloudHook
+from airflow.providers.dbt.cloud.operators.dbt import DbtCloudRunJobOperator
+from airflow.utils.context import Context
```

Comment:
This should be under `TYPE_CHECKING` at least otherwise there will be an implicit, minimum `apache-airflow` requirement of 2.2.3 (when this module was introduced).

Or if https://github.com/astronomer/astronomer-providers/pull/619 is merged before this, you can use the `typing_compat` module the PR introduces.

Reply:
Added condition to check for now, once the https://github.com/astronomer/astronomer-providers/pull/619 is merged it I can use `typing_compat`